saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh

```diff
@@ -253,6 +259,11 @@ resize_partitions() {
 }
 
 main() {
+  if [[ "$SECRET_ENCRYPTION_ENABLED" == "true" ]]; then
+    echo "$SECRET_ENCRYPTION_KEY_SOURCE" 1> /etc/luks/passphrase_encryption_key
```

Comment:
Nit: Out of curiosity, why specify the file descriptor explicitly? `>` by default operates on stdout / file descriptor 1.

Reply:
No particular reason, could be simply `>` I think.